@AH-64D Apache Longbow/Addons/fza_ah64_controls/scripting/functions/weapon/fn_weaponTurretAim.sqf

```diff
@@ -27,7 +27,7 @@ private _inhibit = "";
 private _weaponsProcessorFailed = _heli getVariable "fza_ah64_rwp_fail" && _heli getVariable "fza_ah64_lwp_fail";
 
 private _usingRocket = currentweapon _heli isKindOf["fza_hydra70", configFile >> "CfgWeapons"] || currentWeapon _heli == "fza_rkt_safe";
-private _usingCannon = currentweapon _heli in ["fza_m230", "fza_burstlimiter", "fza_gun_safe"];
+private _usingCannon = currentweapon _heli in ["fza_m230", "fza_burstlimiter"];
```

Comment:
Aha, this was the reason I had multiple safes. However if we base it on `fza_ah64_was` then we don't need it anymore